content/en/docs/tasks/administer-cluster/migrating-from-dockershim/Change-runtime.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Migrating Container runtime from dockershim to containerd"
+weight: 10
+content_type: task 
+---
+
+- Drain Node
+```
+# replace <node-to-drain> with the name of your node you are draining
+kubectl drain <node-to-drain> --ignore-daemonsets
+```
+- Stop docker
+```
+systemctl stop kubelet
+systemctl stop docker
+```
+- Install & start containerd 
+
+
+### For linux systems
+Use the following commands to install Containerd on your system:
+
+Install and configure prerequisites:
```

Comment:
the guide seems to be duplicating steps that already existing the in the "installing a container runtime" page:
https://kubernetes.io/docs/setup/production-environment/container-runtimes/
instead we can cross-link.

the steps can be generalized as:
- drain node
- stop kubelet
- uninstall old CR
- install new CR (CR of your chose, not picking a preference in this page, cross-link to the main CR page)
- update kubelet config and kubeadm bits (optional)
- uncordon node
- check if it works

i know that it works and i've seen users already do it, but before we merge such a guide for hotswap SIG node must tell us if this is even supported and if something unpredictable can happen. cgroup (drivers)? proper cleanup after uninstalling the old CR - logs, container state, etc?

cc @SergeyKanzhelev

Reply:
I think we do want a “dummies guide to” here, because the Docker deprecation is already worrying quite a few people.